Sources/SKSupport/dlopen.swift

```diff
@@ -62,9 +62,11 @@ public struct DLOpenFlags: RawRepresentable, OptionSet {
     public static let deepBind: DLOpenFlags = DLOpenFlags(rawValue: 0)
   #else
     public static let first: DLOpenFlags = DLOpenFlags(rawValue: 0)
+  #if !os(Android)
     public static let deepBind: DLOpenFlags = DLOpenFlags(rawValue: RTLD_DEEPBIND)
```

Comment:
[Same as for SwiftPM](https://github.com/apple/swift-package-manager/pull/2396/files#diff-88191457fc6681ebe42687eda65e4462).